### PR TITLE
Fix cross-build for memcg notification

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Fix the cross-build error from #32577 

Fixes https://github.com/kubernetes/kubernetes/issues/37340

Fixes https://github.com/kubernetes/kubernetes/issues/37383

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37384)

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
fyi @sjenning @rmmh @mikedanese @dchen1107 @ncdc 

I am testing this now, will report back if successful.

**Comment by derekwaynecarr** [comment]
fyi @saad-ali to fix build.

**Comment by derekwaynecarr** [comment]
@pires -- i was able to build on windows.

```
$ KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kubelet
+++ [1123 11:40:10] Generating bindata:
    /home/decarr/go/src/k8s.io/kubernetes/test/e2e/framework/gobindata_util.go
+++ [1123 11:40:11] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [1123 11:40:11] Building go targets for windows/amd64:
    cmd/libs/go2idl/openapi-gen
+++ [1123 11:41:04] Generating bindata:
    /home/decarr/go/src/k8s.io/kubernetes/test/e2e/framework/gobindata_util.go
+++ [1123 11:41:05] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
+++ [1123 11:41:05] Building go targets for windows/amd64:
    cmd/kubelet
```

**Comment by pires** [comment]
Thanks @derekwaynecarr!

**Comment by derekwaynecarr** [comment]
ok, this should go all green now that i updated the test owners file.

**Comment by mikedanese** [comment]
I'd like @rmmh to confirm that cross-builds were failing because of kubelet on windows and not on non amd64 linux archs. I can't find the build output though.

**Comment by rmmh** [comment]
@mikedanese it's in #37340, here's an [example failure](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/kubernetes-cross-build/1778/). It's hard to determine exactly which flavor of builds were failing.

**Comment by rmmh** [comment]
also, you can trigger a cross-build like this:

@k8s-bot build this

**Comment by ncdc** [comment]
That example shows both darwin and windows failures. LGTM if everyone else is ok

**Comment by derekwaynecarr** [comment]
@rmmh -- thanks for the magic invocation on cross-build trigger.  that should clarify things.

**Comment by derekwaynecarr** [comment]
@mikedanese -- should i pull a new version of gazel?

**Comment by mikedanese** [comment]
@derekwaynecarr are you on mac or linux? did you get that diff by running ./hack/update-bazel.sh?

File: pkg/kubelet/eviction/BUILD [hunk 1]
```diff
@@ -16,6 +16,7 @@ go_library(
         "doc.go",
         "eviction_manager.go",
         "helpers.go",
+        "threshold_notifier_unsupported.go",
```
Comment (mikedanese) [inline pkg/kubelet/eviction/BUILD]: This block should only contain files used to build linux, amd64. We don't need this file.

**Comment by mikedanese** [comment]
Every incatation of ./hack/update-bazel.sh should pull a new version of gazel. If you are calling gazel directly then yes please pull new versions fairly often.

**Comment by derekwaynecarr** [comment]
@mikedanese - i was on fedora 24 and got that diff.

**Comment by derekwaynecarr** [comment]
@rmmh @mikedanese -- it looks like cross build is happy again with this, i will try and re-run bazel update to make jenkins happy.

**Comment by derekwaynecarr** [comment]
pushed one last update for bazel, it's possible that diff was from an earlier commit where build tag was wrong on the file.  either way, this should pass cross build per the last version i sent.

@k8s-bot build this

i am not going to be available for next few hours, but this should be good to go.

**Comment by derekwaynecarr** [comment]
Ok.  All the things are happy.

**Comment by pires** [comment]
@ncdc this fixes P0 stuff so is it needed to have P0 label as well?

**Comment by ncdc** [comment]
@pires labeled

**Comment by ncdc** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by spxtr** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by spxtr** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by dchen1107** [comment]
LGTM and I am manually merged this one for build fix.

**Comment by saad-ali** [comment]
Removing cherry-pick candidate label this is already in 1.5

**Pull request merged**